doc/source/ray-observability/state/state-api.rst

```diff
@@ -0,0 +1,303 @@
+.. _state-api-overview-ref:
+
+Monitoring ray states
+=====================
+
+.. tip:: We'd love to hear your feedback on using Ray state APIs - `feedback form <https://forms.gle/gh77mwjEskjhN8G46>`_!
+
+Ray state APIs allow users to conveniently access the current state (snapshot) of Ray through CLI or Python SDK.
+
+.. tip:: APIs are pre-alpha and under active development. APIs are subject to change and not stable across versions.
+
+Getting Started
+---------------
+
+Run any workload. In this example, you will use the following script that runs 2 tasks and creates 2 actors.
+
+.. code-block:: python
+
+    import ray
+    import time
+
+    ray.init(num_cpus=4)
+
+    @ray.remote
+    def task_running_300_seconds():
+        print("Start!")
+        time.sleep(300)
+    
+    @ray.remote
+    class Actor:
+        def __init__(self):
+            print("Actor created")
+    
+    # Create 2 tasks
+    tasks = [task_running_300_seconds.remote() for _ in range(2)]
+
+    # Create 2 actors
+    actors = [Actor.remote() for _ in range(2)]
+
+    ray.get(tasks)
+
+Now, let's see the summarized states of tasks. It takes some time to query all data, so if it doesn't return the output immediately, retry the command.
```

Comment:
With this small workload, it should return quite quickly. Should we mention it is only expected to be slow when querying large number of entities maybe at a later section? 

```suggestion
Now, let's see the summarized states of tasks. 
```